src/compiler/transformers/classFields.ts

```diff
@@ -1341,21 +1685,26 @@ namespace ts {
                         classAliases[getOriginalNodeId(node)] = alias;
                     }
 
-                    // To preserve the behavior of the old emitter, we explicitly indent
-                    // the body of a class with static initializers.
-                    setEmitFlags(classExpression, EmitFlags.Indented | getEmitFlags(classExpression));
-                    expressions.push(startOnNewLine(factory.createAssignment(temp, classExpression)));
+                    expressions.push(factory.createAssignment(temp, classExpression));
+
                     // Add any pending expressions leftover from elided or relocated computed property names
-                    addRange(expressions, map(pendingExpressions, startOnNewLine));
+                    addRange(expressions, pendingExpressions);

```

Comment:
are the removed `startOnNewLine` calls also related to not preserving backward compatibility with the old emitter?

Reply:
Preserved some other way. I moved the `startOnNewLine` call to lines 1703-1708:
```ts
if (expressions.length > 1) {
    // To preserve the behavior of the old emitter, we explicitly indent
    // the body of a class with static initializers.
    addEmitFlags(classExpression, EmitFlags.Indented);
    expressions.forEach(startOnNewLine);
}
```

I may just remove the comment. "old emitter" refers to the emitter pre-transformation which is ancient history at this point. Its more that we put them on new lines to make it more readable when looking at the generated output.